tests/test_bai2.py

```diff
@@ -56,6 +56,15 @@ def test_parse_from_file(self):
             bai2_file = bai2.parse_from_file(f)
             self.assertTrue(isinstance(bai2_file, Bai2File))
 
+    def test_parse_from_file_2(self):
+        from os.path import abspath, join, dirname
+
+        file_path = join(abspath(dirname(__file__)), 'data', 'example2.bai2')
+
+        with open(file_path) as f:
+            bai2_file = bai2.parse_from_file(f)
+            self.assertTrue(isinstance(bai2_file, Bai2File))
+
```

Comment:
What's the difference between `example.bai2` and `example2.bai2`, in terms of the data they contain? Is there something more we can say about what they are/where they're from?